app/assets/javascripts/discourse/app/widgets/quick-access-notifications.js

```diff
@@ -41,6 +41,10 @@ createWidgetFrom(QuickAccessPanel, "quick-access-notifications", {
     return this.getItems().filterBy("read", false).length > 0;
   },
 
+  showDoNotDisturb() {
```

Comment:
What is the point of this method? It always returns true. I see below there's `showDoNotDisturb` but that does not call the method, it just checks if the method exists?

Reply:
Yes, all the panes in this menu are created this way, https://github.com/discourse/discourse/blob/0d6a556c1be23325d5495401c43be5f1d8fe369a/app/assets/javascripts/discourse/app/widgets/quick-access-notifications.js#L5 and the other panes do not have this defined. I moved this to a key/val pair on the notifications panel rather than a function.